packages/lesmis/src/components/Visuals/Chart.js

```diff
@@ -134,19 +119,15 @@ export const Chart = ({
         isLoading={isLoading}
         isError={isError}
         error={error}
-        selectedTab={selectedTab}
+        selectedTab={isExporting ? TABS.CHART : selectedTab}
```

Comment:
fix exporting when table flippa is selected

Reply:
nice!

Reply:
Note the export already always exports the png on web-frontend